PennMobile/Home/Cells/Dining/HomeDiningCell.swift

```diff
@@ -212,15 +232,27 @@ extension HomeDiningCell {
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }
-    
+
+    fileprivate func getSettingsButton() -> UIButton {
+        let button = UIButton()
+        button.setTitleColor(.navigationBlue, for: .normal)
+        button.tintColor = .navigationBlue
+        button.setTitleColor(.secondaryTitleGrey, for: .highlighted)
```

Comment:
Same with this

Reply:
Yes the image is originally black

Reply:
I might be wrong, but I think you can just set tintColor to make the image blue. I'm pretty sure setTitleColor doesn't do anything since the button doesn't have text. Can you try deleting setTitleColor but keeping tintColor and seeing what happens?

Reply:
whoops i read that wrong, will make that change